website/docs/docs/getting-started/2-your-first-test.md

```diff
@@ -0,0 +1,41 @@
+---
+id: your-first-test
+title: Writing your first test
+---
+
+To demonstrate how you can write tests, let's install the TodoMVC app from BigTest:
+```
+$ yarn add -D @bigtest/todomvc
+```
+
+Assuming you went with the default settings for `bigtest.json`, the app command should be configured as `yarn start`:
+```json
+{
+  "app": {
+    "command": "yarn start",
+    "url": "http://localhost:3000"
+  }
+}
+```
+Go into your `package.json` file and modify the start script as `yarn bigtest-todomvc 3000`:
+```json
+{
+  "scripts": {
+    "start": "yarn bigtest-todomvc 3000"
+  }
+}
+```
+Finally, let's go into the `test/` directory and create `todomvc.test.ts`:
+```js
+import { Heading, Page, test } from `bigtest`;
+
+export default test('bigtest todomvc')
+  .step(Page.visit('/'))
+  .assertion(Heading('todos').exists());
+```
+
+That's it! Now you can run `yarn bigtest ci` and you should see one passing step and one passing assertion.
```

Comment:
I'm a bit confused about the relationship between bigtest cli because we had to modify somebody's package json to make a `start`.

Reply:
Between bigtest cli and?

So there were two ways to run todomvc app instead of the reader's app: modify the start script of package.json or modify the app.command in bigtest.json. I chose to go with modifying package.json because we all know how that works.

Once they're done with the tutorial, it'll be easier for someone to realize that the start script needs to be re-adjusted for their own app versus going into bigtest.json and realize that the app.command in bigtest.json is connected to the start script in package.json.

If the guide tells them to change the app.command script to `yarn bigtest-todomvc 3000`, they might think it's a bigtest-related command that needs to stay in their configuration.

Reply:
Ok great info, thanks! clarifies a lot